auth-web/src/composables/affiliations-factory.ts

```diff
@@ -154,6 +155,18 @@ export const useAffiliations = () => {
     )
   }
 
+  /** Returns the Name Request type using the entity type code */
+  const nameRequestType = (business: Business): string => {
+    if (isNameRequest(business)) {
+      const nrType: string = CommonUtils.mapRequestTypeCdToNrType(business.nameRequest?.requestTypeCd)
```

Comment:
You don't need to declare it as a string because Typescript already knows that from the function return type. (And sometimes eslint will complain about a redundant type declaration.)